lib/vsc/eb_hooks/hooks_hydra.py

```diff
@@ -327,4 +345,25 @@ def pre_module_hook(self, *args, **kwargs):  # pylint: disable=unused-argument
         else:
             self.cfg['docurls'] = [usage_info['link']]
 
+    #################################
+    # ------ DUMMY MODULES -------- #
+    #################################
+
+    if IS_CUDA_SOFTWARE:
```

Comment:
Might this be missing a check on being an installation in a non-GPU node? Otherwise we will get dummy modules everywhere:
```suggestion
    if IS_CUDA_SOFTWARE and os.environ['VSC_ARCH_LOCAL'] not in GPU_ARCHS:
```